azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageTypeDaoImpl.java

```diff
@@ -171,4 +235,47 @@ public int createImageType(ImageType imageType) {
       return imageTypes;
     }
   }
+
+  /**
+   * ResultSetHandler implementation class for fetching image type
+   */
+  public static class FetchImageOwnershipHandler implements ResultSetHandler<List<ImageOwnership>> {
+
+    private static final String FETCH_IMAGE_OWNERSHIP_BY_IMAGE_TYPE_NAME =
+        "SELECT it.name, io.id, io.owner, io.role, io.created_by, io.created_on, io.modified_by, "
+            + "io.modified_on FROM image_types it, image_ownerships io  WHERE it.id = io.type_id "
```

Comment:
For any image type, if ownership doesn't exist yet then you should use left outer join. This is applied only when you want to return image type even if ownership doesn't exist.

Reply:
This query is for selecting ownership for the given image type, not for selecting image type metadata.